backend/src/main/java/com/redhat/cloud/notifications/db/EndpointResources.java

```diff
@@ -300,17 +302,17 @@ public void loadProperties(List<Endpoint> endpoints) {
                 .alias("e")
                 .where(
                         WhereBuilder.builder()
-                            .ifElse(
-                                    accountId == null,
-                                    WhereBuilder.builder().and("e.accountId IS NULL"),
-                                    WhereBuilder.builder().and("e.accountId = :accountId", "accountId", accountId)
-                            )
-                            .and(
-                                    WhereBuilder.builder()
-                                            .ifOr(basicTypes.size() > 0, "e.compositeType.type IN (:endpointType)", "endpointType", basicTypes)
-                                            .ifOr(compositeTypes.size() > 0, "e.compositeType IN (:compositeTypes)", "compositeTypes", compositeTypes)
-                            )
-                            .ifAnd(activeOnly != null, "e.enabled = :enabled", "enabled", activeOnly)
+                                .ifElse(
+                                        accountId == null,
+                                        WhereBuilder.builder().and("e.accountId IS NULL"),
+                                        WhereBuilder.builder().and("e.accountId = :accountId", "accountId", accountId)
+                                )
+                                .and(
+                                        WhereBuilder.builder()
+                                                .ifOr(basicTypes.size() > 0, "e.compositeType.type IN (:endpointType)", "endpointType", basicTypes)
+                                                .ifOr(compositeTypes.size() > 0, "e.compositeType IN (:compositeTypes)", "compositeTypes", compositeTypes)
+                                )
+                                .ifAnd(activeOnly != null, "e.enabled = :enabled", "enabled", activeOnly)
```

Comment:
Why do we need extra indentation? 
It just pushes the code to the far right and makes it harder to read.